packages/core/src/components/datetime/test/default/datetime.e2e.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/datetime/test/default/index.html';
+
+test.describe('tds-datetime-default', () => {
+  test('renders datetime component correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('verifies label, helper text, size, and calendar icon for datetime-local component', async ({
+    page,
+  }) => {
+    await page.goto(componentTestPath);
+
+    // Check for the label text
+    const label = page.locator('.tds-datetime-label');
+    const dateTime = page.locator('tds-datetime');
+    const helperText = page.locator('.tds-datetime-helper .tds-helper');
+    const calendarIcon = page.locator('tds-icon[name="calendar"]');
+
+    await expect(dateTime).toHaveAttribute('size', 'lg');
+    await expect(label).toBeVisible();
+    await expect(label).toHaveText('Label text');
+    await expect(helperText).toBeVisible();
+    await expect(helperText).toHaveText('Helper text');
+
+    await expect(calendarIcon).toBeVisible();
+  });
+
+  test('icon click triggers native datetime picker', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Assuming the calendar icon can be clicked to open the datetime picker
+    await page.click('input[type="datetime-local"]');
+
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+
+    // Verify the input is indeed focused. This is an indirect test to ensure
+    // that actions leading up to the display of the native picker are correctly initiated.
+    const inputIsFocused = await page.evaluate(
+      () => document.activeElement.type === 'datetime-local',
+    );
```

Comment:
```suggestion
    const inputIsFocused = await page.evaluate(
  () => (document.activeElement as HTMLInputElement).type === 'datetime-local',
);
```

This line trows an error in my IDE, this seems to fix it.